UNRELEASED.yaml

```diff
@@ -27,4 +30,4 @@
 
 ADDED:
```

Comment:
Ooops looks like I didn't update the changelog after I released yesterday 😂

Reply:
Do you want me to remove this from `UNRELEASED.md` then?

Reply:
Nah we can just have it inserted into the changelog when we do the release 😄

Reply:
👍